content/docs/observability/deployment/online.md

```diff
@@ -121,17 +118,23 @@ To perform an online installation of CSM for Observability, the following steps
     |
     |- Creating namespace karavi                                        Success
     |
+    |- CSI Driver for PowerFlex is installed                            Success
+    |
     |- Copying Secret from vxflexos to karavi                           Success
     |
-    |- Installing CertManager CRDs                                      Success
+    |- CSI Driver for PowerStore is installed                           Success
     |
-    |- Installing Karavi Observability helm chart                       Success
+    |- Copying Secret from powerstore to karavi                         Success
```

Comment:
PowerStore

Reply:
The namespace isn't hardcoded in some of the logged steps. The namespace is stored as a variable so the script prints the namespace that's on the cluster here.